packages/terra-i18n/docs/AggregateTranslations.md

```diff
@@ -1,5 +1,5 @@
 # Aggregating Translations
 
-_The terra-i18n `aggregate-translations` pre-build tool is deprecated._
+_The terra-i18n `aggregate-translations` pre-build tool has been removed from this package._
```

Comment:
Lets remove all docs & link to terra-toolkit. It risk of becoming obsolete (it probably already is).

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2288/commits/6245e266ef54ae055263c4b8679b3c2dca060b12